wadm/mix.exs

```diff
@@ -24,7 +24,7 @@ defmodule Wadm.MixProject do
       {:yaml_elixir, "~> 2.8"},
       {:cloudevents, "~> 0.4.0"},
       {:uuid, "~> 1.1"},
-      {:lattice_observer, git: "https://github.com/wasmcloud/lattice-observer"}
+      {:lattice_observer, git: "https://github.com/janitha09/lattice-observer"}
```

Comment:
Per previous discussion we don't need this one-off, the fix is to make wadm use the `refmap` field, not the `ocimap` field